src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/Blocks/View/addresses.html.twig

```diff
@@ -0,0 +1,70 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="card">
+  <h3 class="card-header">
+    {{ 'Addresses'|trans({}, 'Admin.Global') }}
+    ({{ viewableManufacturer.manufacturerAddresses|length }})
+  </h3>
+  <div class="card-body">
+    {% if viewableManufacturer.manufacturerAddresses is not empty %}
+      {% for address in viewableManufacturer.manufacturerAddresses %}
+        <div class="card">
+          <div class="card-header clearfix">
+            {{ address.first_name }} {{ address.last_name }}
+            <div class="d-inline-block float-right">
+              <a class="btn btn-primary btn-sm"
+                 href="{{ getAdminLink('AdminManufacturers', true, {'id_address': address.id, 'editaddresses': 1}) }}">
+                {{ 'Edit'|trans({}, 'Admin.Actions') }}
+              </a>
+            </div>
+          </div>
```

Comment:
This is a suggestion however this is quite personal, not a best practice: I like to have some blank lines inside big chunks of code such as this one. I find it makes the code easier to read, like a long text split into paragraphs.

Reply:
Agree. :+1: